### PR TITLE
minor formatting

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You must first create an app to get integration key and configure RSA Key Pair a
   :::note
   You'll need to provide RSA private key while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
   :::
-8. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="400" />
+8. Click on **ADD URI** under **Redirect URIs** and enter `http://localhost/` as your new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="400" />
   :::note
   You'll need redirect uri while requesting [application consent](#app-consent).
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:63]:
```suggestion
  You'll need a redirect URI while requesting [application consent](#app-consent).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ You must first create an app to get integration key and configure RSA Key Pair a
 
 ### App Consent
 
-For requesting application consent, follow the steps below:
+Once your app is created you need consent to make API calls. For requesting application consent, follow the steps below:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:70]:
```suggestion
Once your app is created, you need consent to make API calls. To request application consent, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 3]
```diff
@@ -67,7 +67,7 @@ You must first create an app to get integration key and configure RSA Key Pair a
...
 1. Copy the below URL based on your account environment in the browser and press enter, replace **INTEGRATION_KEY**, and **REDIRECT_URI** with the values you copied at step 5 and step 8 of [App](#app) section respectively.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:71]:
```suggestion
1. Copy the below URL based on your account environment in the browser and press enter. Replace **INTEGRATION_KEY** and **REDIRECT_URI** with the values you copied in steps 5 and 8 of the [App](#app) section, respectively.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 4]
```diff
@@ -67,7 +67,7 @@ You must first create an app to get integration key and configure RSA Key Pair a
...
    * For Development (Demo) Environment, `https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI`
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:72]:
```suggestion
   * For the development (demo) environment, use `https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI`
```

**Review by JainM6 — DISMISSED** [review]
Formatting changes.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 5]
```diff
@@ -58,19 +58,19 @@ You must first create an app to get integration key and configure RSA Key Pair a
   :::note
   You'll need to provide RSA private key while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
   :::
-8. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="400" />
+8. Click on **ADD URI** under **Redirect URIs** and enter `http://localhost/` as your new redirect URI. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="400" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:61]:
```suggestion
8. Under **Redirect URIs**, click **ADD URI**, then enter `http://localhost/` as your new redirect URI. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="400" />
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**